### ISSUE TITLE
2.0rc1文档“MNIST数据集使用LeNet进行图像分类”运行时出现警告

### ISSUE THREAD

**Issue by A-Pai** [PaddlePaddle/Paddle#30663]
文档https://www.paddlepaddle.org.cn/documentation/docs/zh/2.0-rc1/tutorial/cv_case/image_classification/image_classification.html
参见项目https://aistudio.baidu.com/aistudio/projectdetail/1441370
中cell：
import numpy as np
import matplotlib.pyplot as plt
train_data0, train_label_0 = train_dataset[0][0],train_dataset[0][1]
train_data0 = train_data0.reshape([28,28])
plt.figure(figsize=(2,2))
plt.imshow(train_data0, cmap=plt.cm.binary)
print('train_data0 label is: ' + str(train_label_0))
运行后出现以下警告，是何故？
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/matplotlib/__init__.py:107: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import MutableMapping
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/matplotlib/rcsetup.py:20: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Iterable, Mapping
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/matplotlib/colors.py:53: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Sized
train_data0 label is: [5]
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/matplotlib/cbook/__init__.py:2349: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  if isinstance(obj, collections.Iterator):
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/matplotlib/cbook/__init__.py:2366: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  return list(data) if isinstance(data, collections.MappingView) else data
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/numpy/lib/type_check.py:546: DeprecationWarning: np.asscalar(a) is deprecated since NumPy v1.16, use a.item() instead
  'a.item() instead', DeprecationWarning, stacklevel=1)

**Comment by vslyu** [comment]
参见https://blog.csdn.net/qq_38125058/article/details/86605232 这个blog不知能否解决你的问题，看Warning消息是版本兼容性问题，推荐升级matplotlib和numpy试试

**Comment by A-Pai** [comment]
> 参见https://blog.csdn.net/qq_38125058/article/details/86605232 这个blog不知能否解决你的问题，看Warning消息是版本兼容性问题，推荐升级matplotlib和numpy试试

这个需要AI Studio平台来升级

**Comment by vslyu** [comment]
> > 参见https://blog.csdn.net/qq_38125058/article/details/86605232 这个blog不知能否解决你的问题，看Warning消息是版本兼容性问题，推荐升级matplotlib和numpy试试
> 
> 这个需要AI Studio平台来升级

好的，收到，会反馈给AI Studio平台的同学排查~
